security/vulnerability-management.md

```diff
@@ -0,0 +1,91 @@
+Vulnerability Management Policy
+===============================
+
+## 1. Purpose
+The purpose of this policy is to establish a standardised approach to identify, assess, and remediate code 
+vulnerabilities in a timely manner. This policy aims to protect our systems, data, and users from potential security 
+threats while minimising the impact on the development process.
+
+## 2. Scope
+This policy applies to all software development projects, including web applications, backend services, etc.  developed 
+or maintained by VisitScotland. It encompasses all stages of the software development lifecycle (SDLC), from initial 
+design through deployment and maintenance.
+
+## 3. Definitions
+- **Vulnerability**: A weakness in the code that can be exploited to compromise the confidentiality, integrity, or 
+  availability of the system.
+- **Remediation**: The process of fixing a vulnerability.
+- **Waive**: Rule to ignore a vulnerability for a specific amount of time
+- **Severity Levels**: Classification of vulnerabilities based on their potential impact.
+
+## 4. Roles and Responsibilities
+- **Developers:** Responsible for writing secure code, conducting initial testing, and fixing identified vulnerabilities.
+- **Project Managers:** Responsible for ensuring that vulnerabilities are addressed and review waivers for 
```

Comment:
Can we have a bit of clarification around this?

Reply:
Hi @jameshaddrillvs,

Excellent spot! It was clear on my head but obviously not in this document I will change that sentence to:

"Responsible to facilitate time for addressing the vulnerabilities, review and validate waivers are sensible or appropriate action is taken"

Does this make more sense?